src/repository/loginDAO.js

```diff
@@ -1,7 +1,17 @@
+const { ScanCommand } = require("@aws-sdk/client-dynamodb");
+const jwt = require("jsonwebtoken");
 const bcrypt = require('bcrypt');
 
+let key = "";
+fs.readFile('./src/key.txt', (err, data) => {
+    if (err) throw err;
+    key = data.toString();
+});
+
+const tableName = "";
+
 async function login(username, password){
```

Comment:
This would be part of a service. Could be a separate login service or it could be included in a user service, but similarly to the middleware, it's main goal is getting a token from JWT and not database interaction so I wouldn't make it a DAO